K8s-deployment/K8s-cluster/addons/mon-stack/prometheus/prometheus-values.yaml

```diff
@@ -720,24 +762,18 @@ server:
   ##
   remoteRead: []
 
+  ## Custom HTTP headers for Liveness/Readiness/Startup Probe
+  ##
+  ## Useful for providing HTTP Basic Auth to healthchecks
+  probeHeaders: []
+
   ## Additional Prometheus server container arguments
   ##
   extraArgs: {}
 
   ## Additional InitContainers to initialize the pod
   ##
-  extraInitContainers: 
-    - name: file-permissions
-      image: busybox:latest
-      command: ['chown', '-R', '65534:65534', '/data/']
-      volumeMounts:
-        - mountPath: /data
-          name: storage-volume
-
-      securityContext:
-        privileged: true
-        runAsUser: 0
-
+  extraInitContainers: []
```

Comment:
Have extraInitContainers has before. this is required for proper  volume permissions

```
extraInitContainers: 
    - name: file-permissions
      image: busybox:latest
      command: ['chown', '-R', '65534:65534', '/data/']
      volumeMounts:
        - mountPath: /data
          name: storage-volume

      securityContext:
        privileged: true
        runAsUser: 0
 ``